src/pages/browse-jobs.html

```diff
@@ -0,0 +1,1390 @@
+                  href="#"
+                  role="button"
+                  data-bs-toggle="dropdown"
+                >
+                  <img
+                    src="../assets/images/search-icon-black-circle.svg"
+                    alt="Search"
+                    class="img-fluid mt-2"
+                  />
+                </a>
+                <ul
+                  class="dropdown-menu dropdown-menu-end search-menu bg-dark"
+                  style="width: 320px"
+                >
+                  <li>
+                    <div
+                      class="d-flex align-items-center bg-white rounded-pill border border-dark mt-2 px-2"
+                    >
+                      <img
+                        src="../assets/images/filter-icon.svg"
+                        alt="Filter"
+                        class="me-2"
+                        style="width: 20px; height: 20px"
+                      />
+                      <div class="border-end me-2" style="height: 20px"></div>
+                      <img
+                        src="../assets/images/search-icon.svg"
+                        alt="Search"
+                        class="me-2"
+                        style="width: 20px; height: 20px"
+                      />
+                      <input
+                        type="text"
+                        class="flex-grow-1 border-0"
+                        placeholder=" Search..."
+                      />
+                      <button type="button" class="btn p-0">
+                        <img
+                          src="../assets/images/close-icon.svg"
+                          alt="Close"
+                          style="width: 15px; height: 15px"
+                        />
+                      </button>
+                    </div>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!--Bottom NavBar Ends-->
+      </nav>
+      <!--End of Navbar-->
+
+      <!-- start of content -->
+      <div class="container-fluid">
+        <!--the image-->
+        <div class="row bg-dark px-5 py-4 mb-3">
+          <h3 class="fw-bold fs-2 text-white pb-2 pt-3 text-center">
+            HRs Magazine Jobs
+          </h3>
+          <div class="col">
+            <div
+              class="card bg-dark text-white landing-img mx-lg-3 mx-md-3 mx-1"
+              style="height: 695px"
+            >
+              <img
+                src="../assets/images/browse-jobs.jpeg"
+                class="card-img rounded-4 image-center"
+                alt="..."
+              />
+            </div>
+          </div>
+        </div>
+        <!--single Card-->
+        <div
+          class="row align-items-center justify-content-center bg-primary mt-4 px-3"
+        >
+          <div class="m-3 bg-light border-0">
+            <div class="row g-0 align-items-center justify-content-center">
+              <nav>
```

Comment:
There is a little bug in the navbar, but i fix it, if you want to see the code pull the updates, Thanks ya Hend